modules/swagger-parser-v3/src/test/java/io/swagger/v3/parser/processors/ExternalRefProcessorTest.java

```diff
@@ -138,4 +140,23 @@ public void testNestedExternalRefs(@Injectable final Schema mockedModel){
     	assertTrue(testedOpenAPI.getComponents().getSchemas().get("Contact") != null);
     	assertTrue(testedOpenAPI.getComponents().getSchemas().get("Address") != null);
     }
+
+	@Test
+	public void testRelativeRefIncludingUrlRef(@Injectable final Schema mockedModel) {
```

Comment:
This requires a server call to `https://raw.githubusercontent.com/`. Please mock it in the unit tests.